drivers/storage/portworx/deployment_test.go

```diff
@@ -702,7 +713,7 @@ func TestPodSpecWithCloudStorageSpec(t *testing.T) {
 		"-max_storage_nodes_per_zone", "2",
 	}
 
-	actual, err = driver.GetStoragePodSpec(cluster)
+	actual, err = driver.GetStoragePodSpec(cluster, nodeName)
```

Comment:
After this we should verify that a StorageNode object is created for `nodeName` node. Also that it has the expected cloud config.

Reply:
how?

Reply:
Just do a get on the fake client and we should find it there because our code creates it.

Reply:
added a separate test for this case